src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -41,14 +60,10 @@ class TableData:
 
     tables: list of pyarrow tables that will joined together for final table yield
     files: list of files that make up tables
-    next_hr_cnt: keeps track of how many files in the next hour have been
-                 processed, when this hits a certain threshold the table
-                 can be yielded
     """
 
-    table: Optional[pyarrow.table] = None
+    table: Optional[pyarrow.Table] = None
     files: List[str] = field(default_factory=list)
-    next_hr_cnt: int = 0
```

Comment:
No longer using `next_hr_cnt` to determine when to yield a table. Table will be yielded any time it goes over 1 million records. This is conservative yielding delimiter and could be increased if needed.